types/truffle-typings/merge.d.ts

```diff
@@ -0,0 +1,52 @@
+/* Generated by ts-generator ver. 0.0.8 */
```

Comment:
Isn't this file now part of builds? I was under the impression we shouldn't been checking in the types that are generated.

Reply:
Ya, I don't know what this is. Sorry. The change was only supposed to be the version bump.